Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.py

```diff
@@ -4439,13 +4468,12 @@ def run_polling_command(client: MsClient, args: dict, cmd: str, action_func: Cal
 
     # distinguish between the initial run, which is the upload run, and the results run
     is_first_run = 'machine_action_id' not in args
-    demisto.debug(f'polling args: {args}')
     if is_first_run:
         command_results = action_func(client, args)
         outputs = command_results.outputs
         # schedule next poll
         polling_args = {
-            'machine_action_id': outputs.get('action_id'),
+            'machine_action_id': outputs.get('action_id') or command_results.raw_response.get("id"),
```

Comment:
Isn't the output extracted from the raw_response? Why would the second work if the first is empty?